ckim105/SER316-Group-2
# Us#9 fix left menu icons

How to Test:
- Run the program and go to the left side panel
- There should be updated icons for each page

 **Developer Checklist**
